packages/decision-tree/src/components/Question/Question.tsx

```diff
@@ -1,26 +1,35 @@
 import React, { FC } from 'react';
 import { type QuestionProps } from './types';
-import { Flex } from '@cmpsr/components';
+import { Flex, Box } from '@cmpsr/components';
 import { Height, SingleChoice, MultipleChoice, Numeric } from './questionTypes';
+import { IDontKnowButton } from './components/IDontKnowButton';
 
 const questionTypesMap = {
-  height: ({ data, answersDispatch }) => <Height data={data} answersDispatch={answersDispatch} />,
-  singleChoice: ({ data, answersDispatch }) => <SingleChoice data={data} answersDispatch={answersDispatch} />,
-  multipleChoice: ({ data, answersDispatch }) => <MultipleChoice data={data} answersDispatch={answersDispatch} />,
-  numeric: ({ data, answersDispatch }) => <Numeric data={data} answersDispatch={answersDispatch} />,
+  height: (props) => <Height {...props} />,
+  singleChoice: (props) => <SingleChoice {...props} />,
+  multipleChoice: (props) => <MultipleChoice {...props} />,
+  numeric: (props) => <Numeric {...props} />,
 };
 
-export const Question: FC<QuestionProps> = ({ data, answersDispatch }) => (
-  <Flex
-    justifyContent="center"
-    flexGrow="1"
-    mt={{ base: 'spacer-8', lg: 'spacer-16' }}
-    mb={{ base: 'spacer-14', lg: 'spacer-16' }}
-    mx={{ base: 'spacer-5', md: 'spacer-20', lg: 'spacer-16', xl: 'spacer-24', xxl: 'spacer-32' }}
-    px={{ base: '0', lg: 'spacer-32', xl: 'spacer-52', xxl: 'spacer-64' }}
-  >
-    {questionTypesMap[data.type]({ data, answersDispatch })}
-  </Flex>
+export const Question: FC<QuestionProps> = ({ submitIDKAnswer, paginationDispatch, ...props }) => (
+  <Box mb={{ base: 'spacer-14', lg: 'spacer-16' }} flexGrow="1">
+    <Flex
+      justifyContent="center"
+      mt={{ base: 'spacer-8', lg: 'spacer-16' }}
+      mb="spacer-8"
+      mx={{ base: 'spacer-5', md: 'spacer-20', lg: 'spacer-16', xl: 'spacer-24', xxl: 'spacer-32' }}
+      px={{ base: '0', lg: 'spacer-32', xl: 'spacer-52', xxl: 'spacer-64' }}
+    >
+      {questionTypesMap[props.data.type](props)}
+    </Flex>
+    {props.data.skippable !== false ? (
+      <Flex justifyContent="center">
+        <IDontKnowButton submitIDKAnswer={submitIDKAnswer} paginationDispatch={paginationDispatch} />
+      </Flex>
+    ) : (
+      ''
+    )}
```

Comment:
is this needed?

Reply:
sadly yes, as I am defaulting to showing the button. Prettier is creating this mess. Odin is working on a PR to make sure all the questions have this property, so then it can be moved back to `??`